packages/server/src/app/services/content/collections.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { PaginateResult } from 'mongoose';
+
+import { ICollections } from '../../shared/content';
+import { JwtPayload } from '@dragonfish/models/auth';
+import { Collection, CollectionForm } from '@dragonfish/models/collections';
+import { CollectionsStore } from '../../db/collections/collections.store';
+
+@Injectable()
+export class CollectionsService implements ICollections {
+    private readonly logger: Logger = new Logger();
+
+    constructor(private readonly collections: CollectionsStore) {}
+
+    async getCollection(user: JwtPayload, collectionId: string): Promise<Collection> {
+        return await this.collections.getOneCollection(user, collectionId);
+    }
+
+    async getAllCollections(user: JwtPayload, pageNum: number): Promise<PaginateResult<Collection>> {
+        return await this.collections.getAllCollections(user, pageNum);
+    }
+
+    async getPublicCollection(userId: string, collectionId: string): Promise<Collection> {
+        return await this.collections.getOnePublicCollection(userId, collectionId);
+    }
```

Comment:
And a service like this is an instance where the distinction between a Store and Service is kinda _eeeeehhhhh_, you know? In some places, you want that extra layer. Here, it's kind of unnecessary.

Reply:
yeah, i was thinking that myself; i can fix it in a later PR